src/internal/signalr/observationshandler.go

```diff
@@ -0,0 +1,187 @@
+package signalr
+
+import (
+	"errors"
+	"math"
+
+	"github.com/futurehomeno/cliffhanger/adapter"
+	"github.com/futurehomeno/cliffhanger/adapter/service/chargepoint"
+	"github.com/futurehomeno/cliffhanger/adapter/service/numericmeter"
+
+	"github.com/futurehomeno/edge-easee-adapter/internal/config"
+)
+
+type ObservationsHandler interface {
```

Comment:
It can be renamed just to `signalr.Handler` as we don't have any other handlers in this package. Also the file can be renamed to `handler.go`